### PR TITLE
updated the README.md

### PR DESCRIPTION
**Pull request by abhraneel2004**
Added the steps to create a feature branch for new users please check and merge. Do comment if something is missing or report any mistake.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,86 @@
...
+=========================================================================================================
+
 <p align="center">
   <img src="https://github.com/Debaditya-Som/Data-Visualization-in-Python/assets/121785700/d40a65d0-ec85-462a-9475-4982a375d067" width=50%/ alt="Data Visualization Image">
 </p>
 
 Data visualization is a powerful tool for understanding and communicating insights from data. This repository aims to provide a comprehensive collection of Python libraries and techniques.
 
+## 001_getting_started
+
```
Comment (Debaditya-Som) [inline README.md:13]: I have created a separate folder for the outputs this part in no longer required

File: README.md [hunk 2]
```diff
@@ -1,10 +1,86 @@
...
+![Visualization](https://github.com/Anushka-Bhowmick/matplotlib/assets/76967222/af4341c1-130b-4df3-a804-c60c68fe1f9a)
+
+## Visualization2
+
+![Visualization2](https://github.com/Anushka-Bhowmick/matplotlib/assets/76967222/15b0b6ea-7a69-4066-9794-e0b0a5c6f81e)
+
+## Ylabel
+
+![Ylabel](https://github.com/Anushka-Bhowmick/matplotlib/assets/76967222/9016c3b8-3866-426d-9cef-1571113daf51)
+
```
Comment (Debaditya-Som) [inline README.md:82]: You may remove this part

**Review by Debaditya-Som — CHANGES_REQUESTED** [review]
Just make the necessary that I have asked you to do, other than that everything is alright

**Comment by abhraneel2004** [comment]
Okay

**Pull request closed without merging**